docs/getting_started/installation/gpu.rocm.inc.md

```diff
@@ -29,48 +29,56 @@ Currently, there are no pre-built ROCm wheels.
     - [PyTorch](https://pytorch.org/)
 
-    For installing PyTorch, you can start from a fresh docker image, e.g, `rocm/pytorch:rocm7.0_ubuntu22.04_py3.10_pytorch_release_2.8.0`, `rocm/pytorch-nightly`. If you are using docker image, you can skip to Step 3.
+    For installing PyTorch and Triton, you can start from a fresh docker image, e.g, `rocm/pytorch:rocm7.0_ubuntu22.04_py3.10_pytorch_release_2.8.0`, `rocm/pytorch-nightly`. If you are using docker image, you can skip to Step 3.
 
     Alternatively, you can install PyTorch using PyTorch wheels. You can check PyTorch installation guide in PyTorch [Getting Started](https://pytorch.org/get-started/locally/). Example:
 
     ```bash
     # Install PyTorch
     pip uninstall torch -y
-    pip install --no-cache-dir torch torchvision --index-url https://download.pytorch.org/whl/nightly/rocm7.0
+    pip install --no-cache-dir torch torchvision --index-url https://download.pytorch.org/whl/rocm6.4
     ```
 
-1. Install [Triton for ROCm](https://github.com/ROCm/triton.git)
+    !!! note
+        For ROCm 7.0 wheels (with gfx950 support), please use the PyTorch nightly build:
+        ```bash
+        pip install --no-cache-dir --pre torch torchvision --index-url https://download.pytorch.org/whl/nightly/rocm7.0
+        ```
 
-    Install ROCm's Triton following the instructions from [ROCm/triton](https://github.com/ROCm/triton.git)
+2. Install [Triton](https://github.com/triton-lang/triton)
+
+    Install Triton on ROCm following the instructions from [ROCm/triton](https://github.com/ROCm/triton):
 
     ```bash
-    python3 -m pip install ninja cmake wheel pybind11
     pip uninstall -y triton
-    git clone https://github.com/ROCm/triton.git
+    git clone https://github.com/ROCm/triton
     cd triton
-    # git checkout $TRITON_BRANCH
-    git checkout f9e5bf54
-    if [ ! -f setup.py ]; then cd python; fi
-    python3 setup.py install
-    cd ../..
+    # git checkout TRITON_BRANCH
+    git checkout 57c693b6
+    pip install -r python/requirements.txt
+    pip install .
+    if [ -d python/triton_kernels ]; then
+        cd python/triton_kernels
+        pip install .
+        cd ../../
+    fi
+    cd ../
     ```
 
     !!! note
         - The validated `$TRITON_BRANCH` can be found in the [docker/Dockerfile.rocm_base](https://github.com/vllm-project/vllm/blob/main/docker/Dockerfile.rocm_base).
         - If you see HTTP issue related to downloading packages during building triton, please try again as the HTTP error is intermittent.
 
-2. Optionally, if you choose to use CK flash attention, you can install [flash attention for ROCm](https://github.com/Dao-AILab/flash-attention.git)
-
-    Install ROCm's flash attention (v2.8.0) following the instructions from [ROCm/flash-attention](https://github.com/Dao-AILab/flash-attention#amd-rocm-support)
+3. Optionally, install [flash attention for ROCm](https://github.com/Dao-AILab/flash-attention)
```

Comment:
We probably shouldn't call it CK flash attention, since that was being called directly in V0 through VLLM_USE_TRITON_FLASH_ATTN=0; we can call it AITER vs Triton FA, or just leave it as optional